actionpack/lib/action_dispatch/request/flash.rb

```diff
@@ -46,7 +46,13 @@ module Flash
 
       extend self
 
-      def use! # :nodoc:
+      # Method to call (`ActionDsiaptch::Request::Flash.use!`) to enable the flash feature in your application.
+      #
+      # Rails applications *not* in API mode are automatically configured, calling this method is not required.
+      # If your application is a Rails API-only one, you can refer to the guide to know how to opt-in.
```

Comment:
I think we should either explain this here or link to the guide

Reply:
Thanks, make sense 👍 . I added some doc